packages/udaru-hapi-16-plugin/test/edgeCases.test.js

```diff
@@ -40,6 +40,7 @@ lab.experiment('Edge cases', () => {
     buildServer({}, err => {
       expect(err).to.exist()
       require.cache[resolved] = null
+      delete require.cache[resolved]
```

Comment:
the previous line is not necessary anymore with the delete

Reply:
agree. will tidy this up